### ISSUE TITLE
Solicitud de charla: Arquitecturas hexagonales y Domain Driven Development

### ISSUE THREAD

**Issue by JonathanLoscalzo** [laplatarb/meetup-nov-2016#7]
Propongo si alguien quiere presentar esto, yo realmente no me animaría por tiempo y no tener una gran experiencia de desarrollo en Ruby o lenguajes del mismo estilo.

Si alguien me ayuda, podríamos hacer un mix, pero me gustaría que alguien apoye esto. 
Ahí van una seríe de cuestiones de leí por arriba de ciertos artículos. 

## Cuestiones
> ¿Por qué Hexagonal?
> ¿Qué tiene que ver con el Domain-Driven Design (DDD)?
> Diferencia con la arquitectura en capas.
> ¿Sirve para aplicaciones Empresariales, relación con Service Bus?
> Relación con Microservices (?)
> Testing.
> Relación con Active Record (Patrón)
> Principios SOLID
> Ok's y Ko's
> ¿Más de lo mismo, sólo un nombre?

## Algunos Links

> http://codely.tv/screencasts/arquitectura-hexagonal-ddd/
> http://www.javiervelezreyes.com/ni-nueva-ni-arquitectura-ni-hexagonal/
> http://culttt.com/2014/12/31/hexagonal-architecture/
> http://laraveles.com/blog/introduccion-arquitectura-hexagonal/
> http://martinfowler.com/articles/badri-hexagonal/

## Cualquier cosa, mis contactos: 
loscalzo.jony@gmail.com
jonathan.r.loscalzo@gmail.com
jloscalzo@hexacta.com
**Skype** Jonathan.Loscalzo

Saludos!

**Comment by F-3r** [comment]
me parece que esto ya está listo! 
2 cosas se me ocurren nomas: 

1) ver si todos los puntos entran en una charla corta de 20/30 min 
2) tener algún caso o experiencia de alguien que haya usado esto en algún sistema en produccion o casi (nosotros tenemos algo así como un anti-caso :P pero no sumaría mucho...)

Si no es para esta meetup, puede ser para la próxima!!

**Comment by JonathanLoscalzo** [comment]
> me parece que esto ya está listo! 

Significa que alguien ya lo hizo!!!!???
Yo lo plantié acá en la empresa para charlarlo pero no se si tuvo mucha repercusión

Gracias!

**Comment by F-3r** [comment]
no, digo, ya armaste toda la estructura, tenés bocha de contenido leído, falta dar la charla nomas :)

**Comment by JonathanLoscalzo** [comment]
ah, yyy si. Pero no me siento capo para dar la charla. 
Pero me vendría bien para practicar. 
Aparte puedo decirle a Christian que me lo tome como final de Ruby ( ? )

**Comment by patriciomacadden** [comment]
2 cosas:
1. No hace falta ser capo para dar una charla, con tener ganas de contar algo dale para adelante!
2. Podés preguntarle si querés, pero no tiene nada que ver con la_plata.rb 😉
